src/ethereum/utils.py

```diff
@@ -0,0 +1,15 @@
+"""
+Utility functions used in this application.
+"""
+
+
+def signum(val: int) -> int:
```

Comment:
Had to look up what `signum` actually. I think something slightly more descriptive like `get_sign`, `calc_sign`, or maybe `sign_bit` would be clearer.

Reply:
update: as usual, sam has a better suggestion on naming